addons/mail/models/mail_mail.py

```diff
@@ -315,16 +318,17 @@ def _prepare_outgoing_body(self):
             return ''
         return self.env['mail.render.mixin']._replace_local_links(self.body_html)
 
-    def _prepare_outgoing_list(self):
+    def _prepare_outgoing_list(self, max_email_size_in_bytes):
```

Comment:
I think it is a bit too much low-level to propagate this parameter ... can't we instead propagate ``mail_server`` from the "send" call, as it makes more sense to me ? Having the outgoing server in the "prepare_outgoing_list" method seems legit for me, what do you think ?

Once we have the mail server we can compute back the email size when necessary.